README.md

```diff
@@ -2,32 +2,89 @@
- - Hosting on GitHub pages (perfect uptime, CDN, Jekyll, custom domains with CNAME)
+There are a few advantages in hosting your API definition on GitHub:
+ - Community engagement (PR's and issues -- if you have a public repo)
  - Advertisment in the GitHub community
+ - Hosting on GitHub pages (perfect uptime, CDN, Jekyll, custom domains with CNAME)
  - Revision history, branching, CI
+ - Review and approval workflows using Pull Requests
  - Fast on-boarding time (developers and tech writers know how to use GitHub :smile:)
  - Fully compatible with Redocly API Reference too
 
+ There are also some advantages to a multi-file YAML format OpenAPI definition:
+ - Reuse schema objects to keep things DRY (don't repeat yourself) 
+ - Smaller diffs compared to JSON (especially for markdown descriptions)
+ - Easier to navigate
+ - Easier to edit with confidence
+
 ## Features
 This generator helps to create a GitHub repo with the following features:
- - Possibility to split a big OpenAPI definition into smaller files and bundle it for deployment
- - Continuous integration/deployment on Travis
+ - Split a big (or small) OpenAPI definition into smaller files organized into folders
+ - Bundle it into a single file for deployment
+ - Continuous integration/deployment on Travis or Redocly Workflows
  - Code samples as separate files
+ - Automate deployment of your OpenAPI definition and docs
  - OpenAPI definition is validated after each commit
- - OpenAPI definition + ReDoc deployed to Github Pages (you can use a custom domain)
- - Live editing in your editor or `swagger-editor` :heart_eyes:
- ![live editing](./live-edit.gif)
+ - Live editing in your editor of choice :heart_eyes:
+
+## Structure
+
+You will have a structure similar to this:
+```
+    ├── LICENSE
+    ├── README.md
+    ├── docs
+    │   ├── favicon.png
+    │   └── index.html
+    ├── openapi
+    │   ├── README.md
+    │   ├── code_samples
+    │   │   ├── C#
+    │   │   │   └── echo
+    │   │   │       └── post.cs
+    │   │   ├── PHP
+    │   │   │   └── echo
+    │   │   │       └── post.php
+    │   │   └── README.md
+    │   ├── components
+    │   │   └── README.md
+    │   └── paths
+    │       └── README.md
+    └── package.json
+```
+
+However, you can adjust it to any structure you prefer.
+
+The `openapi` folder is where your OpenAPI definition will live.  Inside there, and the sub-folders, there are `README.md` files to help guide you further. This is also where your entrypoint `openapi.yaml` will live.
+
+The `components` folder is where you will organize sub-folders such as `schema` to define your schema.
+
+The `paths` folder is where you will organize your paths.  There will be a README.md file in there with suggestions for how to organize it. into specially named files (or folders) that use an `@` in place of a `/` (because files cannot have a `/` character in them).  You will also be able to use path parameters by wrapping them in curly braces `{example}`.
```

Comment:
```suggestion
The `paths` folder is where you will organize your paths. There will be a 'README.md' file in there with suggestions for how to organize it into specially named files (or folders) that use an `@` in place of a `/` (because files cannot have a `/` character in them).  You will also be able to use path parameters by wrapping them in curly braces `{example}`.
```